apps/web/pages/team/[slug].tsx

```diff
@@ -129,7 +129,8 @@ function TeamPage({ team, isUnpublished, markdownStrippedBio, isValidOrgDomain }
                   <span className="text-default text-sm font-bold">{ch.name}</span>
                   <span className="text-subtle block text-xs">
                     {t("number_member", {
-                      count: ch.members.filter((mem) => mem.user.username !== null).length,
+                      count: team.members.filter((mem) => mem.subteams?.includes(ch.slug) && mem.accepted)
+                        .length,
```

Comment:
We now rely on subteam slug matching iterated subteam and accepted property to show members in the profile, as the username will be set even for invited people